HOW-TO.md

```diff
@@ -0,0 +1,14 @@
+# Dependency update
```

Comment:
Thx for doing it! To be reviewed by @immortal-tofu please, later, even if the PR is merged

Reply:
Yes there is also tkms to update to be fully tfhers 1.0.0.
And I can prerelease without merging.

Reply:
> And I can prerelease without merging.

Awesome, but at the same time, it means our release / prerelease are not done on "real commits"?

Reply:
There are real commit but not on main branch (they will be when Clement is back). There's even a prerelease tag to be sure we track them. Is this an issue for you ? A release is often done on release branch BTW not on main, so if you have rc/fix etc it happens outside of main.

Reply:
Eg, what happens if we squash commits? To me, we should do release on things which are merged, not on commits which are still in a PR

Reply:
Merging because other PR need to be added on top of that one.